src/Core/src/Platform/Gtk/RadioButtonExtensions.cs

```diff
@@ -23,78 +19,66 @@ public static void UpdateIsChecked(this RadioButton platformRadioButton, IRadioB
 			"RadioButtonBackgroundDisabled"
 		};
 
-		[MissingMapper]
 		public static void UpdateBackground(this RadioButton platformRadioButton, IRadioButton button)
 		{
-			// if (button.Background is SolidPaint solidPaint)
-			// {
-			// 	UpdateColors(platformRadioButton.Resources, _backgroundColorKeys, solidPaint.ToPlatform());
+			// Ref: https://github.com/jsuarezruiz/maui-linux/blob/ce1c06f1f05422a70c150276a82ad88d3b572198/src/Core/src/Platform/Gtk/ViewExtensions.cs#L15-L69
 
-			// 	platformRadioButton.RefreshThemeResources();
-			// }
-		}
+			var color = button.Background?.BackgroundColor;
 
-		private static readonly string[] _foregroundColorKeys =
-		{
-			"RadioButtonForeground",
-			"RadioButtonForegroundPointerOver",
-			"RadioButtonForegroundPressed",
-			"RadioButtonForegroundDisabled"
-		};
+			if (button.Background is { } paint)
+				color = paint.ToColor();
 
-		[MissingMapper]
-		public static void UpdateTextColor(this RadioButton platformRadioButton, ITextStyle button)
-		{
-			// UpdateColors(platformRadioButton.Resources, _foregroundColorKeys, button.TextColor?.ToPlatform());
+			var gradientCss = button.Background.ToCss();
```

Comment:
i think, you don't need to repeat the code.
the https://github.com/jsuarezruiz/maui-linux/blob/ce1c06f1f05422a70c150276a82ad88d3b572198/src/Core/src/Platform/Gtk/ViewExtensions.cs#L15-L69 extension takes a Gtk.Widget and a IView:

`public static void UpdateBackground(this Widget platformView, IView view)`

as `platformRadioButton` is `RadioButton`, and a `RadioButton` is  a `Widget`, 
and `IRadioButton` is a `IView`
you just need to call:

`platformRadioButton.UpdateBackground(button)`